### ISSUE TITLE
Add debug flag

### ISSUE THREAD

**Issue by rwbogl** [rubik/radon#171]
Radon currently captures most exceptions that occur while doing its work. This makes it a real pain to discover where a programming mistake was made. (See #170 for an example; there's no debug information.)

Could we add a flag or something to avoid suppressing exceptions? Or perhaps just avoid suppressing them altogether?

**Comment by rubik** [comment]
@rwbogl Yes, that would be a good idea. It's not super simple because the error is caught in `radon/__init__.py` and there is no CLI information. A hack would be to check if "-d" or "--debug" is in `sys.argv`, and if so re-raise the exception. That's not a very good solution though.
